packages/commonwealth/client/scripts/views/pages/Community/Groups/common/GroupForm/GroupForm.tsx

```diff
@@ -519,7 +519,8 @@ const GroupForm = ({
             </section>
           </section>
 
-          {formType === 'create' && <TopicGatingHelpMessage />}
+          {formType === 'create' ||
```

Comment:
I'm guessing this should actually be 

```
  {(formType === 'create' || formType === 'edit') && (
      <TopicGatingHelpMessage />
  )}
```

since the banner doesn't appear on the create page anymore.

Reply:
Yes, you're right. Changing it now, thank you.